bin/bundle

```diff
@@ -8,7 +8,7 @@
 # this file is here to facilitate running it.
 #
 
-require "rubygems"
+require 'rubygems'
 
```

Comment:
curious as to what's happening in this bin/bundle file - usually not something we touch in a Rails app so would be curious to hear what function this is serving